docs/develop/howto/faq.md

```diff
@@ -214,9 +218,11 @@ Yes, smart contract addresses are case sensitive because they are generated usin
 
 ### Is it possible to write on Solidity for TON?
 
-  Relatedly, the TON ecosystem does not support development in Ethereum’s Solidity programming language.
+  Recently, the TON ecosystem did not support development in Ethereum’s Solidity programming language.
+  
+  The tool to write TON contracts in Solidity is in development. See it on Github: https://github.com/tonlabs/TON-Solidity-Compiler
```

Comment:
I'm not sure if we should put this in documentation. And even if we should, at least add some warning that this tool is being developed by some 3rd party and we recommend using the languages and tools mentioned in docs.

Reply:
I think there is some chance that eventually this tool will become official, so I think it should be in documentation. Added warning that it is experimental and may work incorrectly.